pkg/controller/oam/applicationconfiguration/applicationconfiguration_test.go

```diff
@@ -20,6 +20,9 @@ import (
 	"context"
 	"testing"
 
+	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	"github.com/crossplane/crossplane-runtime/pkg/test"
```

Comment:
Looks like your editor might have moved import paths to top in a few places

Reply:
I did a code format. The imports are grouped into stdlib/imports/local.